apple/testing/default_runner/macos_test_runner.template.sh

```diff
@@ -149,8 +155,8 @@ fi
 readonly profdata="$TEST_TMP_DIR/coverage.profdata"
 xcrun llvm-profdata merge "$profraw" --output "$profdata"
 
-readonly error_file="$TEST_TMP_DIR/llvm-cov-error.txt"
-llvm_cov_status=0
+readonly export_error_file="$TEST_TMP_DIR/llvm-cov-export-error.txt"
```

Comment:
we can re-use this error file for both, since we check after each to see if it's empty and error, same with the status code. We can rename the variables as needed though